src/hotspot/share/memory/guardedMemory.hpp

```diff
@@ -230,6 +246,20 @@ class GuardedMemory : StackObj { // Wrapper on stack
    */
   void* get_tag() const { return get_head_guard()->get_tag(); }
 
+    /**
```

Comment:
Extra whitespace.  Why these blocky comments?  That say the same thing twice in 5 lines.

Reply:
I am just copying the existing style in this code. It is based on old javadoc style for Java code.

I have fixed the indentation error. Thanks